build_tools/github/doc_environment.yml

```diff
@@ -25,6 +25,7 @@ dependencies:
   - sphinx-gallery
   - numpydoc
   - sphinx-prompt
+  - plotly
```

Comment:
We need to edit the `README.rst` in the root folder to add that `plotly` as a requirement with the minimum associated version as for the other library.